joomla/statistics-server
# Add the initial version of the versions CLI

#### Summary of Changes

As requested by @HLeithner this cli script allows to automate the update of the joomla.json as well as the php.json.

This now also adds PHP pre releases to the mix so we can see whether someone tests on pre releases of PHP.

#### Testing Instructions

run the script cli/cron-update-versions.php and check the json files `/versions/joomla.json` and `/versions/php.json`.

#### Open points

It need to be checked whether the github auth is correctly placed and configured in the prod etc/config.json
